src/core/lib/layer-manager.js

```diff
@@ -237,7 +261,7 @@ export default class LayerManager {
     // render this viewport
     drawLayers(gl, {
       layers: this.layers,
-      viewports: this.context.viewports,
+      viewports: this.getViewports(),
```

Comment:
`getViewports()`, don't we have to pass width/height ? Same for other places we are calling `getViewports()`.